### ISSUE TITLE
Select All Function for Quizzes (Exercises)

### ISSUE THREAD

**Issue by tmccormack165** [EzerIT/BibleOL#14]
The purpose of this objective is to add a button that selects all of the quizzes (exercises) owned by a teacher. There is already an 'Unselect All' button, but there is no 'Select All' button. 

To view the quizzes panel:
```
1. Log onto Bible Online Learner as a adminstrator
2. Go to Administration >> Manage Exercises
```

These steps will take you to this page:
<img width="1220" alt="Screenshot 2023-10-03 at 10 04 27 PM" src="https://github.com/EzerIT/BibleOL/assets/60988784/86fcde7c-f51e-4463-999b-d8a9346d2a0e">

In the top left corner of the exercises table you will see an Unselect All button, but no Select All button, the objective of this task is to add a functional Select All button.

**Comment by tmccormack165** [comment]
Check All Button:
<img width="1069" alt="Screenshot 2023-11-12 at 7 05 12 PM" src="https://github.com/EzerIT/BibleOL/assets/60988784/d25f3c16-4eed-4f44-82de-4f61a6ffb841">

**Comment by tmccormack165** [comment]
This issue has been solved by adding an almost identical checkAll() function to the uncheckAll() function.
```
function uncheckAll() {
      $('input[name="file[]"]:checked').prop('checked',false);
    }



function checkAll() {
      $('input[name="file[]"]:not(:checked)').prop('checked',true);
    }
```

The issue is closed with this commit: f3aae78db292defde335e706889117d2f814a22e
